Sound/Tidal/Stream.hs

```diff
@@ -218,6 +218,17 @@ infixl 1 |/|
 (|/|) :: ParamPattern -> ParamPattern -> ParamPattern
 (|/|) = mergeNumWith (div) (/)
 
+{- | These are shorthand for merging lists of patterns with @#@, @|*|@, @|+|@,
```

Comment:
Just a note: you can see if an operator has been "taken" in haskell by searching it on Hayoo (http://hayoo.fh-wedel.de/) -- for operators, enclose them in parens like ```(++)```